javascript-conditionals/main.js

```diff
@@ -35,31 +35,31 @@ console.log('Does it start with J:', startsWithJ(string));
 
 const person = {
   name: 'Andrew Ortiz',
-  Age: 27,
+  age: 27,
 };
 function isOldEnoughToDrink(person) {
-  if (person >= 21) {
-    return false;
-  } else {
+  if (person.age >= 21) {
     return true;
+  } else {
+    return false;
   }
 }
-console.log('old enough to drink:', isOldEnoughToDrink(person));
+console.log('is old enough to drink', isOldEnoughToDrink(person));
 
 function isOldEnoughToDrive(person) {
-  if (person >= 16) {
-    return false;
-  } else {
+  if (person.age >= 16) {
     return true;
+  } else {
+    return false;
   }
 }
 console.log('is old enough to drive: ', isOldEnoughToDrive(person));
 
 function isOldEnoughToDrinkAndDrive(person) {
-  if (person < 21) {
-    return true;
-  } else {
+  if (person.age >= 16 && person.age >= 21) {
     return false;
+  } else {
+    return true;
```

Comment:
This should be `return false`. We only want to `return false` from this function